src/modules/infobox/components/FeatureCombo.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+    <div class="edit-feature" data-infobox="height-reference">
+        <div class="edit-feature-form">
+            <FeatureStyleEdit
+                :feature="feature"
+                :available-icon-sets="availableIconSets"
+                @change:title="onTitleChange"
+                @change:description="onDescriptionChange"
+                @change:text-size="onTextSizeChange"
+                @change:text-color="onTextColorChange"
+                @change:color="onColorChange"
+                @change:icon="onIconChange"
+                @change:icon-size="onIconSizeChange"
+            />
+        </div>
+
+        <FeatureProfile class="edit-feature-plot" :feature="feature" />
+    </div>
+</template>
+
+<script>
+import { mapActions, mapState } from 'vuex'
+import { EditableFeature } from '@/api/features.api'
+import FeatureStyleEdit from './styling/FeatureStyleEdit.vue'
+import FeatureProfile from './FeatureProfile.vue'
+
+export default {
+    components: {
+        FeatureStyleEdit,
+        FeatureProfile,
+    },
+    props: {
+        feature: {
+            type: EditableFeature,
+            required: true,
+        },
+    },
+    computed: {
+        ...mapState({
+            availableIconSets: (state) => state.drawing.iconSets,
+        }),
+    },
+    methods: {
+        ...mapActions([
+            'changeFeatureTitle',
+            'changeFeatureDescription',
+            'changeFeatureColor',
+            'changeFeatureTextSize',
+            'changeFeatureTextColor',
+            'changeFeatureIcon',
+            'changeFeatureIconSize',
+        ]),
```

Comment:
As many parts of the Vue code is similar between the 3 different flavour of Feature***.vue I would maybe ask you to try to either use a mixin, but that is not the recommended way to do in Vue3 anymore, so maybe have a look at https://vuejs.org/guide/reusability/composables.html#composables and see if you can make it happen here 😉